data/json/items/migration.json

```diff
@@ -149,5 +149,11 @@
     "type": "MIGRATION",
     "replace": "jar_glass_sealed",
     "contents": [ "tomato_canned" ]
+  },
+  {
+    "id": "jar_kompot",
```

Comment:
> That's a pretty minor detail that's rather tedious to keep up with manually.

The alternative is merge conflicts when multiple authors update the file which is definitely worse. The cost should be paid by the committer as opposed to future maintainers.


> Is there a tool that can do it automatically?

`jq` but you'll then need to reformat. I'd just copy and paste for the number of entries you have involved. I keep considering writing an automated formatter but thats well beyond the scope of this PR